hc/front/tests/test_add_channel.py

```diff
@@ -30,6 +36,7 @@ def test_it_trims_whitespace(self):
         self.assertEqual(q.count(), 1)
 
     def test_instructions_work(self):
+        """Test that instructions for each channel integration work while adding"""
```

Comment:
double quotes detected. Previous comments use single quotes. Why the change?. Its good practice to stick to one convention.